hmpps-kotlin-spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -140,3 +143,10 @@ project.getTasksByName("check", false).forEach {
   }
   it.dependsOn("$prefix:ktlintCheck")
 }
+
+dependencyCheck {
```

Comment:
copy across configuration from the dps gradle spring boot project as it doesn't get applied automatically